generators/server/templates/src/main/java/package/config/SecurityConfiguration.java.ejs

```diff
@@ -241,9 +244,9 @@ public class SecurityConfiguration extends WebSecurityConfigurerAdapter {
         .and()
             .headers()
             <%_ if (clientTheme !== 'none') { _%>
-            .contentSecurityPolicy("default-src 'self'; frame-src 'self' data:; script-src 'self' 'unsafe-inline' 'unsafe-eval' https://storage.googleapis.com; style-src 'self' https://fonts.googleapis.com 'unsafe-inline'; img-src 'self' data:; font-src 'self' https://fonts.gstatic.com data:")
+            .contentSecurityPolicy(environment.getProperty("security.headers.content-security-policy","default-src 'self'; frame-src 'self' data:; script-src 'self' 'unsafe-inline' 'unsafe-eval' https://storage.googleapis.com; style-src 'self' https://fonts.googleapis.com 'unsafe-inline'; img-src 'self' data:; font-src 'self' https://fonts.gstatic.com data:"))
             <%_ } else { _%>
-            .contentSecurityPolicy("default-src 'self'; frame-src 'self' data:; script-src 'self' 'unsafe-inline' 'unsafe-eval' https://storage.googleapis.com; style-src 'self' 'unsafe-inline'; img-src 'self' data:; font-src 'self' data:")
+            .contentSecurityPolicy(environment.getProperty("security.headers.content-security-policy","default-src 'self'; frame-src 'self' data:; script-src 'self' 'unsafe-inline' 'unsafe-eval' https://storage.googleapis.com; style-src 'self' 'unsafe-inline'; img-src 'self' data:; font-src 'self' data:"))
```

Comment:
Are you sure this works? Does it require a `spring.` prefix?

Reply:
Actually this is the old property name which was used by spring for the very same purpose,   and now it is deprecated  in favor of in-code configuration.  

This in aplication.yml:
----------------------%<---------
#  Uncomment to override generated content security policy
security:
       headers:
           content-security-policy: "default-src 'self'; frame-src 'self' data:; script-src 'self' 'unsafe-inline' 'unsafe-eval' https://storage.googleapis.com; style-src 'self' https://fonts.googleapis.com 'unsafe-inline'; img-src 'self' data: https://*.tile.openstreetmap.org ; font-src 'self' https://fonts.gstatic.com data:"

--------------%<-----------------
now definitely produces this:

-------------------------%<------------------------
default-src 'self'; frame-src 'self' data:; script-src 'self' 'unsafe-inline' 'unsafe-eval' https://storage.googleapis.com; style-src 'self' https://fonts.googleapis.com 'unsafe-inline'; img-src data: 'self' data: https://*.tile.openstreetmap.org ; font-src 'self' https://fonts.gstatic.com data:
------------------------%<------------------

Reply:
However -  there may  be better choice for property name to separate it from old spring  name -  which is flagged y IntelliJ as obsolete.

Reply:
I would prefer a dedicated (jhipster)property to avoid confusion with the old spring name

Reply:
Played around with property name name -  JHipsterProperties does not like unknown property names.  this would require change in  java library also.

Reply:
I you like you can do the change in https://github.com/jhipster/jhipster-bom too. We have already `jhipster.security` and could e.g. add a csp property. We should just agree on a property name

Reply:
I would  say - just change kebab case to camel .  but it will require yet another pull request

Reply:
Added pull request to BOM:

https://github.com/jhipster/jhipster-bom/pull/235

Reply:
Original branch also pushed